.azure/build_pipeline.yaml

```diff
@@ -0,0 +1,34 @@
+# Triggers
+trigger:
+  branches:
+    include:
+      - 'main'
+      - 'release-*'
+  tags:
+    include:
+      - '*'
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - '*'
+
+jobs:
+  - job: 'main_branch_build'
+    displayName: 'Build'
+    # Set timeout for jobs
+    timeoutInMinutes: 60
+    # Base system
+    pool:
+      vmImage: 'Ubuntu-18.04'
+    # Pipeline steps
+    steps:
+      - bash: make prepare
+        displayName: "Download kafka"
+      - bash: make docker_build
+        displayName: "Build images"
+      - bash: make docker_tag_push
+        displayName: "Tag and push images"
+        env:
+          QUAY_USER: $(QUAY_USER)
+          QUAY_PASS: $(QUAY_PASS)
```

Comment:
You do not want to run this from PRs ... normally only from main branch and tags. I do not see any handling for it here.

Reply:
Changed to main and only tags